core/assets/bundles/bundle_vi.properties

```diff
@@ -558,34 +558,35 @@ sector.lost = Khu vực [accent]{0}[white] đã mất!
 #note: the missing space in the line below is intentional
 sector.captured = Khu vực [accent]{0}[white]đã chiếm!
 
-threat.low = Thấp
-threat.medium = Trung bình
-threat.high = Cao
-threat.extreme = Cực cao
-threat.eradication = Hủy diệt
+threat.low = Độ nguy hiểm thấp
+threat.medium = Độ nguy hiểm trung bình
+threat.high = Độ nguy hiểm cao
+threat.extreme = Độ nguy hiểm cực cao
+threat.eradication = Độ nguy hiểm hủy diệt
```

Comment:
Ingame be like
English: Threat: Low
Vietnamese: Mối đe dọa: Độ nguy hiểm thấp ("too long ... and ... i think you can understand what i want to say ?")